ldap/3.4/modules/ROOT/pages/index.adoc

```diff
@@ -5,550 +5,47 @@ Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#
 
 Anypoint Connector for Lightweight Directory Access Protocol (LDAP) is a public standard that facilitates maintenance and access to distributed directories (such as network user privilege information) over an Internet Protocol (IP) network.
 
-Many LDAP servers are included in free-use open source projects and packages. The particulars of installing, running and configuring LDAP servers fall outside the scope of this document.
+LDAP Connector supports both LDAP over SSL and LDAP over TLS.
 
-This document provides an example of a simple LDAP Connector configuration, including basic instructions for installing and connecting to an LDAP server using LDAP Connector.
+For an SSL connection use any port other than 636.
```

Comment:
@dhirsch1 , this statement seems incorrect. SSL connection uses 636 by default.

Reply:
I will remove the sentence.